### PR TITLE
Feature/spring boot native

### PR DESCRIPTION
**Pull request by rdehuyss**

### CODE REVIEW

**Comment by rdehuyss** [comment]
Keep open ..

File: core/src/main/java/org/jobrunr/utils/ClassPathUtils.java [hunk 1]
```diff
@@ -68,6 +68,8 @@ private static Path toPath(URL url) {
             }
             if ("jar".equals(uri.getScheme())) {
                 return JarFileSystemUtils.toPath(uri);
+            } else if ("resource".equals(uri.getScheme())) {
```
Comment (kibertoad) [inline core/src/main/java/org/jobrunr/utils/ClassPathUtils.java:71]: why not just use switch?

File: core/src/main/java/org/jobrunr/utils/ResourceFileSystemUtils.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            FileSystem fs = getFileSystem();
+            return fs.getPath(StringUtils.substringAfter(uri.toString(), ":"));
+        } catch (ProviderNotFoundException e) {
+            throw new ProviderNotFoundException("Provider not found for URI " + uri.toString());
+        }
+    }
+
+    private static FileSystem getFileSystem() throws IOException {
+        if (fileSystem == null) {
+            HashMap<String, Object> options = new HashMap<>();
```
Comment (kibertoad) [inline core/src/main/java/org/jobrunr/utils/ResourceFileSystemUtils.java:33]: `Map` should be sufficient for the left side

File: settings.gradle [hunk 3]
```diff
@@ -7,8 +15,9 @@ include ':language-support:jobrunr-kotlin-16-support'
 include ':framework-support:jobrunr-micronaut-feature'
 include ':framework-support:jobrunr-quarkus-extension:deployment'
 include ':framework-support:jobrunr-quarkus-extension:runtime'
-include ':framework-support:jobrunr-quarkus-extension:tests'
+//include ':framework-support:jobrunr-quarkus-extension:tests'
```
Comment (kibertoad) [inline settings.gradle:18]: should this be commented out?

**Comment by rdehuyss** [comment]
Merged in feature/v5 with resolved feedback.

**Pull request closed without merging**